cmd/antrea-agent/util.go

```diff
@@ -79,3 +84,25 @@ func parsePortRange(portRangeStr string) (start, end int, err error) {
 
 	return start, end, nil
 }
+
+func getServiceListOptionsModifier(serviceProxyName string) (func(*metav1.ListOptions), error) {
+	var serviceNameSelector, nonHeadlessServiceSelector *labels.Requirement
```

Comment:
Better to define `serviceNameSelector` as `serviceWithProxyNameSelector`

Reply:
Will do.